meta/fraction-metadata/src/main/java/org/wildfly/swarm/fractions/FractionUsageAnalyzer.java

```diff
@@ -284,6 +312,13 @@ private void loadDetectorsAndScanners() {
         detectorsLoaded = true;
     }
 
+    private String transformClassSource(String entry) {
+        entry = entry.replace("WEB-INF/classes/", "");
+        entry = entry.replace(".class", "");
+        entry = entry.replace(File.separator, ".");
```

Comment:
I'm just thinking -- did you try this on Windows? I'm not sure what separators will be used e.g. for .zip entries...?